We can predict whether a given gene will be expressed or not by considering the effects of histone marks by moving a sliding window across the genome. The same coordinates on the genome can exhibit both epigenetic marks of activation and repression at different levels. Thus, predicting these modificationsâ€™ net effect on gene expression forms an interesting and challenging machine-learning problem. In this project, we consider the effect of the following modifications: H3K4me3 (activator), H3K4me1 (repressor), H3K36me3 (indicates activation), H3K27me3 (repressor) and H3K9me3 (repressor) on the expression of multiple genes. Marker types are associated with promoter regions, enhancer regions, transcribed regions, polycomb repression, and heterochromatin regions. We use different machine-learning algorithms to predict the effect of histone modification markers on gene expression.

Please see the report for a detailed discussion about the models developed, results and figures!
